README.rst

```diff
@@ -75,21 +74,14 @@ are listed in the table below.
    C         `bmi-c`_        `bmi-example-c`_
    C++       `bmi-cxx`_      `bmi-example-cxx`_
    Fortran   `bmi-fortran`_  `bmi-example-fortran`_
+   Java      `bmi-java`_     `bmi-example-java`_
    Python    `bmi-python`_   `bmi-example-python`_
    ========  ==============  ======================
 
 Detailed instructions for building the specifications and examples
 are given at each link above.
-The specifications can also be installed through `conda`.
 
-.. code-block:: bash
```

Comment:
Why did you remove the install instructions?

Reply:
More detailed explanations are given with each of the specs and examples, so this is redundant. And the Java spec and example uses Maven.

As a compromise, how about this:
> Alternatively, the specifications can be installed through conda (C, C++, Fortran, Python) or Maven (for Java). See the links above for details.

Reply:
👍